sympy/printing/maple.py

```diff
@@ -120,9 +120,9 @@ def _print_Pow(self, expr, **kwargs):
         PREC = precedence(expr)
         if expr.exp == -1:
             return '1/%s' % (self.parenthesize(expr.base, PREC))
-        elif expr.exp == 0.5 or expr.exp == S(1) / 2:
+        elif expr.exp in (0.5, S.Half):
```

Comment:
These lines are pointless. Floats compare equal to exactly equal rationals:

```py
>>> 0.5 == S(1)/2
True
```

This could just be `expr.exp == S.Half` and so on.

Reply:
OK! Didn't try it, although I reflected upon it... I'll change it.

Reply:
Ideally they wouldn't compare equal (#20033)